wasm/jsapi/constructor/multi-value.any.js

```diff
@@ -0,0 +1,148 @@
+// META: global=jsshell
+// META: script=/wasm/jsapi/wasm-module-builder.js
+// META: script=/wasm/jsapi/assertions.js
+
+promise_test(async () => {
+  const type_if_fi = makeSig([kWasmF32, kWasmI32], [kWasmI32, kWasmF32]);
```

Comment:
```suggestion
  const type_if_fi = makeSig([kWasmI32, kWasmF32], [kWasmF32, kWasmI32]);
```

Confims with name and usage line 22: `result.instance.exports.swap(7, 4.2);`

Reply:
Also type_if_fi is incorrectly defined multiple times in the file, could we hoist it?

Reply:
Good catch! Swapped the arguments instead, and moved to f64 to avoid rounding errors.